spec/models/user_spec.rb

```diff
@@ -1,39 +0,0 @@
-require 'rails_helper'
-
-describe User do
```

Comment:
How about we use the `it_behaves_like "a gds-sso user class"` example - https://github.com/alphagov/gds-sso#testing-in-your-application?